src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -33,6 +39,8 @@ internal class VsCodeWindowManager : IVsCodeWindowManager, IVsCodeWindowEvents
 
             private IDisposable? _navigationBarController;
             private IVsDropdownBarClient? _dropdownBarClient;
+            private ElementHost? _documentOutlineViewHost;
+            private CancellationTokenSource? _documentOutlineCts;
```

Comment:
```suggestion
            private CancellationTokenSource? _documentOutlineTokenSource;
```

We don't generally abbreviate unless for extremely common proper names (e.g. Xml).